otx/algorithms/segmentation/adapters/mmseg/datasets/dataset.py

```diff
@@ -118,6 +118,7 @@ def __init__(
         pipeline: Sequence[dict],
         classes: Optional[List[str]] = None,
         test_mode: bool = False,
+        **kwargs,
```

Comment:
Why is this being added?

Reply:
By updated `OTXTask`, `data_dir`, `ann_dir` and some other things income dataset class. classification, detection dataset class also have `**kwargs`.